retirement_api/templates/retirement_api/claiming.html

```diff
@@ -408,16 +412,16 @@ <h3 class="lifestyle-question">{% trans questions.longlife.question %}</h3>
             </div>
           </div>
           <div class="lifestyle-response" data-responds-to="yes">
-            <h4>{% trans questions.longlife.answer_yes_a_subhed %}</h4>
-            <p>{% trans questions.longlife.answer_yes_a %}</p>
+            <h4>{% trans questions.longlife.answer_yes_a_subhed|safe %}</h4>
```

Comment:
You might space out the pipe so it stands out a bit better 
```
<h4>{% trans questions.longlife.answer_yes_a_subhed | safe %}</h4>
```

Here and below.

Reply:
Ah, good point. I kept the spacing consistent with the other `|safe` bits above that line, but I'm happy to change them all if adding the spaces fits better with our styles.

Reply:
In general I think it does, but we're not consistent with it.

Reply:
Huh, I just tried to add the space around the pipe, and I get a `TemplateSyntaxError` if there are spaces. So apparently no spaces it is.

Reply:
Very strange!

Reply:
Jinja2 is more forgiving about pipe filters, although the [docs](https://jinja.palletsprojects.com/en/2.10.x/templates/#filters) suggest using no spaces.
Django templates don't allow a space.